presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import static java.util.Objects.requireNonNull;
+
+public class StatisticsValue<T>
+{
+    private final Reliability reliability;
+    private final T value;
+
+    public StatisticsValue(Reliability reliability, T value)
+    {
+        this.reliability = requireNonNull(reliability, "reliability can not be null");
+        this.value = value;
```

Comment:
I dropped min/max for now. We will resume discussion when we have usecases for that.